dyno-core/src/main/java/com/netflix/dyno/connectionpool/impl/MonitorConsole.java

```diff
@@ -159,6 +159,7 @@ public TokenPoolTopology getTopology(String cpName) {
             config.put("socketTimeout", String.valueOf(cpConfig.getSocketTimeout()));
             config.put("timingCountersResetFrequencyInSecs",
                     String.valueOf(cpConfig.getTimingCountersResetFrequencySeconds()));
+            config.put("replicationFactor", String.valueOf(pool.getTopology().getReplicationFactor()));
```

Comment:
This new property is only used in one place and at the same time, I also see the code calculates the RF dynamically.  Would there be a wrong logic later on in setting this as well as automatic calculation?